### DISCUSSION TITLE
Sharing The Money App With Multiple Nextcloud Users

### DISCUSSION THREAD

**Discussion by phantomanalytical** [powerpaul17/nc_money#46] [Q&A]
Hello,

Perhaps this is a Nextcloud setting that I don't know about, but I'm wanting to share this app with my wife's Nextcloud account so we can manage our money together. I have both of our accounts added to a group (in case that's necessary for what I'm trying to do). How can I go about syncing our financial input in this app with each other without being on the same account?

**Comment by powerpaul17** [comment]
Unfortunately sharing of information is up to the apps themselves and currently there is no such feature implemented. There is already a request in the issues but I haven't got the time yet to get to it.

**Comment by myurkoski** [comment]
I am excited about this feature for the exact same reason as the original post. This app has really changed our ability to dynamically and strategically manage funds in our household, as we are in very lean times right now. Having my spouse be able to see and interact with the data asynchronously allows us to further build upon good habits!
